web/src/lib/llm/utils.ts

```diff
@@ -34,6 +34,34 @@ export function getFinalLLM(
   return [provider, model];
 }
 
+export function getLLMProviderForPersona(
```

Comment:
This function appears to check specifically for an override, so perhaps renaming the function with respect to this would be more descriptive.